### ISSUE TITLE
2025-04-18, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#144]
# Today's Arxiv Papers

(1) [Aligning Constraint Generation with Design Intent in Parametric CAD](http://arxiv.org/abs/2504.13178v1), 2025-04-17

(2) [Sleep-time Compute: Beyond Inference Scaling at Test-time](http://arxiv.org/abs/2504.13171v1), 2025-04-17

(3) [Exploring Expert Failures Improves LLM Agent Tuning](http://arxiv.org/abs/2504.13145v1), 2025-04-17

(4) [Syntactic and Semantic Control of Large Language Models via Sequential Monte Carlo](http://arxiv.org/abs/2504.13139v1), 2025-04-17

(5) [Energy-Based Reward Models for Robust Language Model Alignment](http://arxiv.org/abs/2504.13134v1), 2025-04-17

(6) [LLMs Meet Finance: Fine-Tuning Foundation Models for the Open FinLLM Leaderboard](http://arxiv.org/abs/2504.13125v1), 2025-04-17

(7) [Low-hallucination Synthetic Captions for Large-Scale Vision-Language Model Pre-training](http://arxiv.org/abs/2504.13123v1), 2025-04-17

(8) [VistaDPO: Video Hierarchical Spatial-Temporal Direct Preference Optimization for Large Video Models](http://arxiv.org/abs/2504.13122v1), 2025-04-17

(9) [Uncertainty-Aware Trajectory Prediction via Rule-Regularized Heteroscedastic Deep Classification](http://arxiv.org/abs/2504.13111v1), 2025-04-17

(10) [EventVAD: Training-Free Event-Aware Video Anomaly Detection](http://arxiv.org/abs/2504.13092v1), 2025-04-17

(11) [Retrieval-Augmented Generation with Conflicting Evidence](http://arxiv.org/abs/2504.13079v1), 2025-04-17

(12) [SkyReels-V2: Infinite-length Film Generative Model](http://arxiv.org/abs/2504.13074v1), 2025-04-17

(13) [Early Accessibility: Automating Alt-Text Generation for UI Icons During App Development](http://arxiv.org/abs/2504.13069v1), 2025-04-17

(14) [Accuracy is Not Agreement: Expert-Aligned Evaluation of Crash Narrative Classification Models](http://arxiv.org/abs/2504.13068v1), 2025-04-17

(15) [RoboTwin: Dual-Arm Robot Benchmark with Generative Digital Twins](http://arxiv.org/abs/2504.13059v1), 2025-04-17

(16) [Aspect-Based Summarization with Self-Aspect Retrieval Enhanced Generation](http://arxiv.org/abs/2504.13054v1), 2025-04-17

(17) [GraphAttack: Exploiting Representational Blindspots in LLM Safety Mechanisms](http://arxiv.org/abs/2504.13052v1), 2025-04-17

(18) [How Large Language Models Are Changing MOOC Essay Answers: A Comparison of Pre- and Post-LLM Responses](http://arxiv.org/abs/2504.13038v1), 2025-04-17

(19) [InstructRAG: Leveraging Retrieval-Augmented Generation on Instruction Graphs for LLM-Based Task Planning](http://arxiv.org/abs/2504.13032v1), 2025-04-17

(20) [ChatEXAONEPath: An Expert-level Multimodal Large Language Model for Histopathology Using Whole Slide Images](http://arxiv.org/abs/2504.13023v1), 2025-04-17

(21) [SHA256 at SemEval-2025 Task 4: Selective Amnesia -- Constrained Unlearning for Large Language Models via Knowledge Isolation](http://arxiv.org/abs/2504.12996v1), 2025-04-17

(22) [Chain-of-Thought Prompting for Out-of-Distribution Samples: A Latent-Variable Study](http://arxiv.org/abs/2504.12991v1), 2025-04-17

(23) [A Virtual Machine for Arbitrary Low-Precision GPGPU Computation in LLM Serving](http://arxiv.org/abs/2504.12984v1), 2025-04-17

(24) [Accommodate Knowledge Conflicts in Retrieval-augmented LLMs: Towards Reliable Response Generation in the Wild](http://arxiv.org/abs/2504.12982v1), 2025-04-17

(25) [Sparks of Science: Hypothesis Generation Using Structured Paper Data](http://arxiv.org/abs/2504.12976v1), 2025-04-17

(26) [Estimating Optimal Context Length for Hybrid Retrieval-augmented Multi-document Summarization](http://arxiv.org/abs/2504.12972v1), 2025-04-17

(27) [QLLM: Do We Really Need a Mixing Network for Credit Assignment in Multi-Agent Reinforcement Learning?](http://arxiv.org/abs/2504.12961v1), 2025-04-17

(28) [Are Retrials All You Need? Enhancing Large Language Model Reasoning Without Verbalized Feedback](http://arxiv.org/abs/2504.12951v1), 2025-04-17

(29) [Customizing Emotional Support: How Do Individuals Construct and Interact With LLM-Powered Chatbots](http://arxiv.org/abs/2504.12943v1), 2025-04-17

(30) [Explainable AI in Usable Privacy and Security: Challenges and Opportunities](http://arxiv.org/abs/2504.12931v1), 2025-04-17

(31) [ConExion: Concept Extraction with Large Language Models](http://arxiv.org/abs/2504.12915v1), 2025-04-17

(32) [MAIN: Mutual Alignment Is Necessary for instruction tuning](http://arxiv.org/abs/2504.12913v1), 2025-04-17

(33) [Benchmarking Multi-National Value Alignment for Large Language Models](http://arxiv.org/abs/2504.12911v1), 2025-04-17

(34) [Information Gain-Guided Causal Intervention for Autonomous Debiasing Large Language Models](http://arxiv.org/abs/2504.12898v1), 2025-04-17

(35) [EmoVoice: LLM-based Emotional Text-To-Speech Model with Freestyle Text Prompting](http://arxiv.org/abs/2504.12867v1), 2025-04-17

(36) [DashChat: Interactive Authoring of Industrial Dashboard Design Prototypes through Conversation with LLM-Powered Agents](http://arxiv.org/abs/2504.12865v1), 2025-04-17

(37) [Can LLMs reason over extended multilingual contexts? Towards long-context evaluation beyond retrieval and haystacks](http://arxiv.org/abs/2504.12845v1), 2025-04-17

(38) [Assesing LLMs in Art Contexts: Critique Generation and Theory of Mind Evaluation](http://arxiv.org/abs/2504.12805v1), 2025-04-17

(39) [EarthGPT-X: Enabling MLLMs to Flexibly and Comprehensively Understand Multi-Source Remote Sensing Imagery](http://arxiv.org/abs/2504.12795v1), 2025-04-17

(40) [Enhancing the Geometric Problem-Solving Ability of Multimodal LLMs via Symbolic-Neural Integration](http://arxiv.org/abs/2504.12773v1), 2025-04-17

(41) [GraphOmni: A Comprehensive and Extendable Benchmark Framework for Large Language Models on Graph-theoretic Tasks](http://arxiv.org/abs/2504.12764v1), 2025-04-17

(42) [Trajectory Adaptation using Large Language Models](http://arxiv.org/abs/2504.12755v1), 2025-04-17

(43) [Chinese-Vicuna: A Chinese Instruction-following Llama-based Model](http://arxiv.org/abs/2504.12737v1), 2025-04-17

(44) [Pandora: A Code-Driven Large Language Model Agent for Unified Reasoning Across Diverse Structured Knowledge](http://arxiv.org/abs/2504.12734v1), 2025-04-17

(45) [Validating LLM-Generated Relevance Labels for Educational Resource Search](http://arxiv.org/abs/2504.12732v1), 2025-04-17

(46) [SimUSER: Simulating User Behavior with Large Language Models for Recommender System Evaluation](http://arxiv.org/abs/2504.12722v1), 2025-04-17

(47) [SmartFreeEdit: Mask-Free Spatial-Aware Image Editing with Complex Instruction Understanding](http://arxiv.org/abs/2504.12704v1), 2025-04-17

(48) [Collaborative Perception Datasets for Autonomous Driving: A Review](http://arxiv.org/abs/2504.12696v1), 2025-04-17

(49) [Why and How LLMs Hallucinate: Connecting the Dots with Subsequence Associations](http://arxiv.org/abs/2504.12691v1), 2025-04-17

(50) [Data-efficient LLM Fine-tuning for Code Generation](http://arxiv.org/abs/2504.12687v1), 2025-04-17

(51) [WebLists: Extracting Structured Information From Complex Interactive Websites Using Executable LLM Agents](http://arxiv.org/abs/2504.12682v1), 2025-04-17

(52) [GRAIL: Gradient-Based Adaptive Unlearning for Privacy and Copyright in LLMs](http://arxiv.org/abs/2504.12681v1), 2025-04-17

(53) [Persona-judge: Personalized Alignment of Large Language Models via Token-level Self-judgment](http://arxiv.org/abs/2504.12663v1), 2025-04-17

(54) [Scaling Instruction-Tuned LLMs to Million-Token Contexts via Hierarchical Synthetic Data Generation](http://arxiv.org/abs/2504.12637v1), 2025-04-17

(55) [Towards Characterizing Subjectivity of Individuals through Modeling Value Conflicts and Trade-offs](http://arxiv.org/abs/2504.12633v1), 2025-04-17

(56) [Code Copycat Conundrum: Demystifying Repetition in LLM-based Code Generation](http://arxiv.org/abs/2504.12608v1), 2025-04-17

(57) [GeoSense: Evaluating Identification and Application of Geometric Principles in Multimodal Reasoning](http://arxiv.org/abs/2504.12597v1), 2025-04-17

(58) [Efficient MAP Estimation of LLM Judgment Performance with Prior Transfer](http://arxiv.org/abs/2504.12589v1), 2025-04-17

(59) [Identifying and Mitigating the Influence of the Prior Distribution in Large Language Models](http://arxiv.org/abs/2504.12585v1), 2025-04-17

(60) [Provable Secure Steganography Based on Adaptive Dynamic Sampling](http://arxiv.org/abs/2504.12579v1), 2025-04-17

(61) [Prompt-Driven and Training-Free Forgetting Approach and Dataset for Large Language Models](http://arxiv.org/abs/2504.12574v1), 2025-04-17

(62) [MetaSynth: Meta-Prompting-Driven Agentic Scaffolds for Diverse Synthetic Data Generation](http://arxiv.org/abs/2504.12563v1), 2025-04-17

(63) [ZeroSumEval: Scaling LLM Evaluation with Inter-Model Competition](http://arxiv.org/abs/2504.12562v1), 2025-04-17

(64) [CDF-RAG: Causal Dynamic Feedback for Adaptive Retrieval-Augmented Generation](http://arxiv.org/abs/2504.12560v1), 2025-04-17

(65) [Benchmarking LLM-based Relevance Judgment Methods](http://dx.doi.org/10.1145/3726302.3730305), 2025-04-17

(66) [ELAB: Extensive LLM Alignment Benchmark in Persian Language](http://arxiv.org/abs/2504.12553v1), 2025-04-17

(67) [Memorization: A Close Look at Books](http://arxiv.org/abs/2504.12549v1), 2025-04-17

(68) [Knowledge Acquisition on Mass-shooting Events via LLMs for AI-Driven Justice](http://arxiv.org/abs/2504.12545v1), 2025-04-17

(69) [Memorization vs. Reasoning: Updating LLMs with New Knowledge](http://arxiv.org/abs/2504.12523v1), 2025-04-16

(70) [Evaluating the Diversity and Quality of LLM Generated Content](http://arxiv.org/abs/2504.12522v1), 2025-04-16

(71) [Multimodal LLM Augmented Reasoning for Interpretable Visual Perception Analysis](http://arxiv.org/abs/2504.12511v1), 2025-04-16

(72) [Can Pre-training Indicators Reliably Predict Fine-tuning Outcomes of LLMs?](http://arxiv.org/abs/2504.12491v1), 2025-04-16

(73) [Towards Conversational AI for Human-Machine Collaborative MLOps](http://arxiv.org/abs/2504.12477v1), 2025-04-16

(74) [Integrating Structural and Semantic Signals in Text-Attributed Graphs with BiGTex](http://arxiv.org/abs/2504.12474v1), 2025-04-16

(75) [SLURG: Investigating the Feasibility of Generating Synthetic Online Fallacious Discourse](http://arxiv.org/abs/2504.12466v1), 2025-04-16

(76) [PlanGlow: Personalized Study Planning with an Explainable and Controllable LLM-Driven System](http://arxiv.org/abs/2504.12452v1), 2025-04-16

(77) [Position: The Most Expensive Part of an LLM should be its Training Data](http://arxiv.org/abs/2504.12427v1), 2025-04-16

(78) [Don't Just Translate, Agitate: Using Large Language Models as Devil's Advocates for AI Explanations](http://dx.doi.org/10.5281/zenodo.15170455), 2025-04-16

(79) [Mitigating LLM Hallucinations with Knowledge Graphs: A Case Study](http://arxiv.org/abs/2504.12422v1), 2025-04-16

(80) [A Human-AI Comparative Analysis of Prompt Sensitivity in LLM-Based Relevance Judgment](http://dx.doi.org/10.1145/3726302.3730159), 2025-04-16

(81) [Activated LoRA: Fine-tuned LLMs for Intrinsics](http://arxiv.org/abs/2504.12397v1), 2025-04-16

(82) [BitNet b1.58 2B4T Technical Report](http://arxiv.org/abs/2504.12285v1), 2025-04-16

(83) [HLS-Eval: A Benchmark and Framework for Evaluating LLMs on High-Level Synthesis Design Tasks](http://arxiv.org/abs/2504.12268v1), 2025-04-16

(84) [FLIP Reasoning Challenge](http://arxiv.org/abs/2504.12256v1), 2025-04-16

(85) [AnomalyGen: An Automated Semantic Log Sequence Generation Framework with LLM for Anomaly Detection](http://arxiv.org/abs/2504.12250v1), 2025-04-16

(86) [MOS: Towards Effective Smart Contract Vulnerability Detection through Mixture-of-Experts Tuning of Large Language Models](http://arxiv.org/abs/2504.12234v1), 2025-04-16

(87) [Watermarking Needs Input Repetition Masking](http://arxiv.org/abs/2504.12229v1), 2025-04-16

(88) [d1: Scaling Reasoning in Diffusion Large Language Models via Reinforcement Learning](http://arxiv.org/abs/2504.12216v1), 2025-04-16

(89) [Themisto: Jupyter-Based Runtime Benchmark](http://arxiv.org/abs/2504.12365v1), 2025-04-16

(90) [From Requirements to Architecture: Semi-Automatically Generating Software Architectures](http://arxiv.org/abs/2504.12192v1), 2025-04-16

(91) [What Do Large Language Models Know? Tacit Knowledge as a Potential Causal-Explanatory Structure](http://dx.doi.org/10.1017/psa.2025.19), 2025-04-16

(92) [SALAD: Improving Robustness and Generalization through Contrastive Learning with Structure-Aware and LLM-Driven Augmented Data](http://arxiv.org/abs/2504.12185v1), 2025-04-16

(93) [Trusting CHATGPT: how minor tweaks in the prompts lead to major differences in sentiment classification](http://arxiv.org/abs/2504.12180v1), 2025-04-16

(94) [ARCeR: an Agentic RAG for the Automated Definition of Cyber Ranges](http://arxiv.org/abs/2504.12143v1), 2025-04-16

(95) [Multilingual Contextualization of Large Language Models for Document-Level Machine Translation](http://arxiv.org/abs/2504.12140v1), 2025-04-16

(96) [Clarifying Ambiguities: on the Role of Ambiguity Types in Prompting Methods for Clarification Generation](http://arxiv.org/abs/2504.12113v1), 2025-04-16

(97) [Towards LLM Agents for Earth Observation](http://arxiv.org/abs/2504.12110v1), 2025-04-16

(98) [Entropy-Guided Watermarking for LLMs: A Test-Time Framework for Robust and Traceable Text Generation](http://arxiv.org/abs/2504.12108v1), 2025-04-16

(99) [Gauging Overprecision in LLMs: An Empirical Study](http://arxiv.org/abs/2504.12098v1), 2025-04-16

(100) [Reasoning-Based AI for Startup Evaluation (R.A.I.S.E.): A Memory-Augmented, Multi-Step Decision Framework](http://arxiv.org/abs/2504.12090v1), 2025-04-16
